learning-development/accessibility.qmd

```diff
@@ -4,33 +4,67 @@ title: "Accessibility"
 
 <p class="text-muted">Some quick links and tips for increasing awareness of accessibility in digital content</p>
 
+
+---
+
+## Accessibility in DfE
+
+DfE have our own [Digital Accessibility Hub](https://educationgovuk.sharepoint.com/sites/lvewp00043) filled with helpful information on making the digital world more accessible.
+
+There are also the [DfE design manual](https://design.education.gov.uk), which has been created by the digital teams at DfE and includes [DfE's digital accessibility guidance](https://design.education.gov.uk/accessibility).
+
+---
+
+## Colour accessibility tools
+
+Contrast:
+
+- [WebAim (comparing two colours)](https://webaim.org/resources/contrastchecker/)
+- [Colour contrast matrix (comparing multiple colours](https://jansensan.github.io/color-contrast-matrix/)
+
+Colour blindness:
+
+- [Adobe colour accessibility (gives a pass / fail)](https://color.adobe.com/create/color-accessibility)
```

Comment:
This doesn't work on desktop for me

Reply:
Weird, looks like just adding a '/' at the end fixes it, not sure how I got the wrong link though will push the fix shortly

Reply:
Made this change in my PR comments commit